app/containers/CreateImportWallet/ImportWallet/ImportWallet.jsx

```diff
@@ -0,0 +1,139 @@
+// @flow
+import React from 'react'
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs'
+import { ROUTES } from '../../../core/constants'
+import CloseButton from '../../../components/CloseButton'
+import BackButton from '../../../components/BackButton'
+import Button from '../../../components/Button'
+import CreateImportWalletForm from '../../../components/CreateImportWalletForm'
+import FullHeightPanel from '../../../components/Panel/FullHeightPanel'
+import { loadWalletRecovery } from '../../Settings/Settings'
+import ImportIcon from '../../../assets/icons/import.svg'
+import AddIcon from '../../../assets/icons/add.svg'
+import styles from './ImportWallet.scss'
+
+type Props = {
+  // eslint-disable-next-line react/no-unused-prop-types
+  authenticated: boolean,
+  // eslint-disable-next-line react/no-unused-prop-types
+  setAccounts: (Array<Object>) => any,
+  // eslint-disable-next-line react/no-unused-prop-types
+  showSuccessNotification: Object => any,
+  // eslint-disable-next-line react/no-unused-prop-types
+  showErrorNotification: Object => any,
+}
+
+type State = {
+  tabIndex: number,
+}
+
+const IMPORT_KEY_OPTIONS = {
+  WIF: {
+    render: (props: Props) => (
+      <CreateImportWalletForm
+        option="IMPORT"
+        importKeyOption="WIF"
+        authenticated={props.authenticated}
+      />
+    ),
+    display: 'Private Key',
+  },
+  ENCRYPTED_WIF: {
+    render: (props: Props) => (
+      <CreateImportWalletForm
+        option="IMPORT"
+        importKeyOption="ENCRYPTED_WIF"
+        authenticated={props.authenticated}
+      />
+    ),
+    display: 'Encrypted Key',
+  },
+
+  RECOVER_WALLET: {
```

Comment:
Can we add some instructions here to fill out the empty space?
```
  RECOVER_WALLET: {
    render: (props: Props) => (
      <Fragment>
        <p>
          Upload a JSON wallet recovery file here to add your accounts to Neon.
          This option is also available on the Settings page.
        </p>
        <div
          className={styles.buttonContainer}
          onClick={() => {
            loadWalletRecovery(
              props.showSuccessNotification,
              props.showErrorNotification,
              props.setAccounts,
            )
          }}
        >
          <Button
            renderIcon={AddIcon}
            type="submit"
            shouldCenterButtonLabelText
            primary
          >
            Import File
          </Button>
        </div>
      </Fragment>
    ),
    display: 'Recover Wallet',
  },```

Reply:
<img width="618" alt="Screen Shot 2019-07-02 at 11 43 58 AM" src="https://user-images.githubusercontent.com/13072035/60526710-d99a5400-9cbe-11e9-8421-7d9a503cdab0.png">

Reply:
Nice suggestion! I will make the changes once we have clarified the point above related to the `eslint` warning.